docs/.vitepress/config.ts

```diff
@@ -93,66 +93,92 @@ export default defineConfig({
+                // { text: 'Remove a follow', link: '/developers/guides/advanced/todo' },
+                // { text: 'Update profile data', link: '/developers/guides/advanced/todo' },
+                // { text: 'Reset profile data', link: '/developers/guides/advanced/todo' },
+                { text: 'Create Ethereum verification ', link: '/developers/guides/advanced/verify-address' },
+                { text: 'Submit casts to the hub', link: '/developers/guides/hub-scripts/casts' }
+              ],
+            },
+            {
+              text: 'Building apps',
               collapsed: false,
               items: [
-                { text: 'Create casts with embedded content', link: '/developers/guides/advanced/adv-casts' },
-                { text: 'Create a cast in a channel', link: '/developers/guides/advanced/create-channel-casts' },
-                { text: 'Change your custody address', link: '/developers/guides/advanced/transfer-fid' },
-                { text: 'Change your farcaster name', link: '/developers/guides/advanced/transfer-fname' },
-                { text: 'Change your recovery address', link: '/developers/guides/advanced/change-recovery' },
-                { text: 'Fetch casts from a channel', link: '/developers/guides/advanced/fetch-channel-casts' },
-                { text: 'Create an address verification ', link: '/developers/guides/advanced/verify-address' },
+                { text: 'Replicate to Postgres', link: '/developers/guides/basics/replicate' },
+                // { text: 'Generate a chronological feed for a user', link: '/developers/guides/applications/feed' },
+                // { text: 'Create a bot to post messages to the hub', link: '/developers/guides/applications/bot' },
+                // { text: 'Build a farcaster client', link: '/developers/guides/applications/client' },
+                // { text: 'Read data with Farcaster Connect', link: '/developers/guides/applications/todo' },
+                // { text: 'Write data with Farcaster Connect', link: '/developers/guides/applications/todo' },
+              ],
+            },
+            {
+              text: 'Advanced',
+              collapsed: true,
+              items: [
                 {
                   text: 'Counting signups by day',
                   link: '/developers/guides/advanced/query-signups',
                 },
                 { text: 'Decode key metadata', link: '/developers/guides/advanced/decode-key-metadata' },
-                { text: 'Add a signer', link: '/developers/guides/advanced/add-signer' },
-                { text: 'Register a user', link: '/developers/guides/advanced/register-a-user' },
-                { text: 'Lookup account by name', link: '/developers/guides/advanced/name-lookup' },
-                // { text: 'How do I roll a signer?', link: '/developers/guides/applications/todo' },
+                // { text: 'Compact links', link: '/developers/guides/advanced/todo' },
```

Comment:
@sanjayprabhu and @horsefacts added a couple more ideas for tutorials here that we can flesh out